File: DSA/Data_Structure/Arrays/Easy/Plus_One/Solutions/solve.js
var plusOne = function(digits) {
    let len = digits.length;
    for(let i = len - 1; i >=0; i--){
        if(digits[i] === 9){
            if(i === 0){
                digits[i] = 1;
                digits[len] = 0;
            } else {
                digits[i] = 0;
            }
        } else {
            digits[i] = digits[i] + 1;
            break;
        }
    }
    return digits;
};

File: DSA/Data_Structure/Arrays/Easy/Plus_One/Solutions/solve.py
def plusOne(digits):
    return list(map(int, list(str(int(''.join(map(str, digits))) + 1))))
    
def main():
    print(plusOne([6,1,4,5,3,9,0,1,9,5,1,8,6,7,0,5,5,4,3]))

main()